src/libraries/System.Private.CoreLib/src/System/TimeSpan.cs

```diff
@@ -291,6 +291,23 @@ public int CompareTo(object? value)
 
         public static TimeSpan FromDays(double value) => Interval(value, TicksPerDay);
 
+        public static TimeSpan FromDays(int days)
```

Comment:
>       public static TimeSpan FromDays(int days) [](http://example.com/codeflow?start=1&length=48)

need to add all Xml docs to all added APIs.

Reply:
Yeah 👍. I am trying to look at the failing test. But having problems getting a
Ok bits to debug it on mac.

Reply:
>  I am trying to look at the failing test. 

I think you need to modify the test: https://github.com/dotnet/runtime/blob/19dafb6855790ccf24e3781c85c43f180c3f0b82/src/libraries/System.Linq.Expressions/tests/DebugViewTests.cs#L319.

You need to add a line like:

```C#
ParameterExpression l = Expression.Parameter(typeof(long), "l");
```

then change:

```diff
- Check(".Call System.TimeSpan.FromSeconds($d)", Expression.Call(null, typeof(TimeSpan).GetMethod("FromSeconds", new[] { typeof(int) }), d));
= Check(".Call System.TimeSpan.FromSeconds($l)", Expression.Call(null, typeof(TimeSpan).GetMethod("FromSeconds", new[] { typeof(int) }), l));
```